index.html

```diff
@@ -14,7 +14,7 @@
         <h1>Cool Rocks</h1>
     </header>
     <main>
-        <a href ="../products.html">Click here for cool rocks</a>
+        <a href="../products">Click here for cool rocks</a>
     </main>
 
     <script src="app.js"></script>
```

Comment:
Delete this line. this file doesn't exist and your index html doesn't require any javascript anyways.